src/kanata/mod.rs

```diff
@@ -972,6 +972,29 @@ impl Kanata {
                                 }
                             }
                         }
+                        CustomAction::LiveReloadFile(st_path) => {
```

Comment:
`st_path` is unusual naming. I would rename it to `path` and shadow the variable on line 976.

```rust
                        CustomAction::LiveReloadFile(path) => {
                            let path = PathBuf::from(path);
```